lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1669,6 +1669,32 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
       return b.create<arith::MulIOp>(loc, lhs, rhs);
     }
   }
+  if (auto gtTensor = dyn_cast<AtenGtTensorOp>(op)) {
+    AtenGtTensorOp::Adaptor adaptor(operands);
+    Type lhsDtype = payloadArgs[0].getType();
+    Type rhsDtype = payloadArgs[1].getType();
+
+    Type elementalType =
+        gtTensor.self().getType().cast<BaseTensorType>().getDtype();
+
+    if (lhsDtype != rhsDtype)
+      gtTensor.emitError("unimplemented: different lhs and rhs dtype");
```

Comment:
Add a TODO here to handle type promotion.

Reply:
Done.